packages/common/types/passport.ts

```diff
@@ -0,0 +1,4 @@
+export type Passport = {
```

Comment:
Is this shared with the access bridge codebase?

Reply:
Yes, but made this branch from Develop so it's missing here, but later they will be merged. It's in the same place.